lib/expressions.js

```diff
@@ -140,6 +206,28 @@ expressions.rotate = {
     var v = new THREE.Vector3(this.axis[0],this.axis[1],this.axis[2]);
     this.matrix.makeRotationAxis(v, this.angle);
   },
+  bounds: function() {
+    // find max distance to rotation axis, expand cube to that *sqrt(2) around axis.
+    // this could likely be optimized, but calc trade-off?
+    if (this.angle === 0) {
+      // trivial
+      return this.operands[0].bounds();
+    }
+    // TODO: introduce rotation center rotation? what if this.origin is not 0?
```

Comment:
Let's leave the center of rotation at the origin.  Other centers can be a translation followed by rotation followed by translation back.